_rails/how-to-guides/deployment/parallel-deployment.md

```diff
@@ -103,20 +103,23 @@ In this strategy we deploy to each server one at a time. We take each server off
 In this strategy we deploy to all your server simultaneously. Servers are not taken off the load balancer. If your webserver does not support zero-downtime reloads you will most likely have some downtime while your application reloads.
 
 ### Rolling deployment
-In order to use rolling deployments, your application needs: 
+
+In order to use rolling deployments, your application needs:
 
 - A load balancer
-- At least four servers (not including the load balancer)...
-- ...two of which **must be web servers**
+- At least four servers (not including the load balancer)…
+- …two of which **must be web servers**
 
 Rolling deployments work as follows:
 
-1. Your servers are split as evenly as possible into two groups
-2. The first group will be taken off the load balancer, deployed to in parallel, then added back to load balancer
+1. Your servers are split into two groups based on a heuristic that takes factors like server size into account. The groups are balanced by server load and role (as far as possible).
+2. The first group is taken off the load balancer, deployed to in parallel, then added back to the load balancer
 3. If the first group deploys successfully, the second group will then be taken off the load balancer, deployed to in parallel, then added back
 4. If either group deployment fails, it will not be added back to load balancer and deployment will stop.
 5. When deploying again, the previously failed group (if present) will always go first
 
+You can [tag your application servers](/{{page.collection}}/how-to-guides/deployment/tagging-components.html) as either `c66.deployment.group.first` or `c66.deployment.group.last` to indicate your preference for ordering. We will abide by this ordering where possible (but cannot guarantee it).
```

Comment:
The first and last group tag were changed to `c66.deployment.group.sapphire` and `c66.deployment.group.emerald`

Reply:
@jin-cloud66 Thanks Jing! I've updated the doc now. Could you have a look at the new commit?